2022/azurecgn/data.json

```diff
@@ -9,7 +9,7 @@
     "Name": "Azure Meetup Cologne",
     "Country": "Germany",
     "Url": "https://www.rakoellner.de/global-azure-cgn/",
-    "SessionizeApiId": "xrmtfv0v",
```

Comment:
Hello @christianwaha ! When you browse to this url with your ApiId it should return valid json. There is something not correctly configured in your Sessionize! Please take a look at that and resolve! /Magnus

Reply:
Hi Magnus I thought I pushed a version without the API key, maybe you can remove it and push it to be visible on the website?